### ISSUE TITLE
OSCCondition.cpp / 299 / CheckCondition(): Story board element "event_follow_trajectory3

### ISSUE THREAD

**Issue by Qianyu-Liu** [esmini/esmini#95]
Hi,
I am running esmini on Ubuntu with python (version 3.6.9).
The terminal keeps printing  "OSCCondition.cpp / 299 / CheckCondition(): Story board element "event_follow_trajectory3" not found" 

Do you have any idea that why I am getting this ?
![Screenshot from 2021-03-31 13-43-14](https://user-images.githubusercontent.com/64209360/113139643-ad12c380-9227-11eb-9002-7a136b58aa44.png)

**Comment by Qianyu-Liu** [comment]
Thanks for the information. It was the typo mistake on scenario file.
And if there is any possibility to turn off all the printing on terminal from esmini, just want to save some computing time.
![Screenshot from 2021-04-02 21-15-12](https://user-images.githubusercontent.com/64209360/113446975-41805000-93f9-11eb-9bb9-fd8bb90ae296.png)

**Comment by eknabevcc** [comment]
Good to hear that you found the condition error yourself. 

Then, thanks for the suggestion to remove log messages to console. We realize there are many useless messages, many remaining from early days when we just used esmini for learning and parsing OpenSCENARIO. A cleaning job should have been done, but it's boring. But I found a little bit of motivation now that you asked for it.

So now we're done two things:
1. Heavily reduced number of messages 
2. Added an option to prevent log messages to console/stdout

The changes have been [committed to master](https://github.com/esmini/esmini/commit/035ee7e1006b0611227ba22f3451a4a9244113a2) and will be part of next release. 👍

We'll keep this issue open until you have tested the option.

**Comment by Qianyu-Liu** [comment]
Thanks so much!
I am using the shared library  'libesminiLib.so'  in python. Can I get the updated file somewhere in github? or do I need wait for the next release ? If yes, do you have any time plan for next release.
Thanks again!

**Comment by eknabevcc** [comment]
Here's an updated lib that you can test (until next release):
https://www.dropbox.com/s/6yqdpaxdogwrdsw/libesminiLib.so?dl=1

You will immediately experience the reduced messages. However in order to completely disable it you can either 

1. Initialize esmini with arguments (using SE_InitWithArgs) or 
2. Call the SE_LogToConsole(false); before SE_Init. E.g:

```
#include "esminiLib.hpp"

int main(int argc, char* argv[])
{
	SE_LogToConsole(false);
	SE_Init("../resources/xosc/cut-in.xosc", 0, 1, 0, 0);

	for (int i = 0; i < 500; i++)
	{
		SE_Step();
	}

	return 0;
}
```

**Comment by eknabevcc** [comment]
Updates released since long (2021-04-13 Version 2.7.0). Closing issue.
